### ISSUE TITLE
<Box /> responsiveBreakpoint not functional

### ISSUE THREAD

**Issue by atanasster** [grommet/grommet#2827]
changing the value of responsiveBreakpoint to anything different from 'small' generating crash.

desired behavior is to implement the borderSize, edgeSize for `medium` and `large` in base theme, similar to already implemented `small` https://github.com/grommet/grommet/blob/d6c3a7f83c9fc3dea641eafac16ac87ba2121890/src/js/themes/base.js#L124

```
const theme = {
  box: {
    responsiveBreakpoint: "medium"
  }
};

    <Grommet theme={theme}>
      <Box background="brand" pad="large" />
    </Grommet>

```
code sandbox
https://codesandbox.io/s/pj8175227j

**Comment by oorestisime** [comment]
Hm in the end not sure if this is a bug. 
The theme defines the edgeStyles only for small https://github.com/grommet/grommet/blob/master/src/js/themes/base.js#L121-L154 so if you define medium breakpoint you need to handle this values as well.

I see two solutions here:

- add edgestyle etc for each breakpoint but this beats the logic that if you define xlarge or tablet or.. breakpoint then you will consider this again as a bug. 
- Grommet could console.warn that you try to access theme properties that are not set in order to have a better readable error.

Thoughts?

**Comment by atanasster** [comment]
I would suggest this is a bug and add the edgeSize etc. for `medium` - since this breakpoint is defined in `base.js`, it is not defined by the user.

large might not really make sense as it's the largest breakpoint, xlarge etc. custom added breakpoints are responsability of users that do add those breakpoints.

**Comment by oorestisime** [comment]
Hm, i am probably missing something but not even sure what is the purpose they serve in the small breakpoint. 
how are the values going to change in the medium size? and how are those values different from `global.edgeSize`, `global.borderSize`?

**Comment by atanasster** [comment]
@oorestisime the current use of `global.edgeSize`, `global.borderSize` is when responsive is false and also in several cases where it's hard-coded, while the small breakpoint - when Box turns into the `column` direction from `row-responsive`.
Imho it would be best to have those spacings/borders adapt not only when the breakpoint is switched, but for all sizes depending on the current screen size.
For example use the `medium` sizes when the screen is in that size range, even if we have a responsiveBreakpoint of `small`.
For now, just to avoid crashing, I think either add the exact same values to `medium` as already defined for `small`, or have checks in the code to avoid the crashing.

**Comment by jcfilben** [comment]
This issue was fixed in #4644 so I think this issue can be closed
